src/sodiumCompatibility/java/net/coderbot/iris/compat/sodium/mixin/vertex_format/MixinSodiumWorldRenderer.java

```diff
@@ -21,6 +24,6 @@
                             ")Lme/jellysquid/mods/sodium/client/render/chunk/ChunkRenderBackend;"))
     private ChunkVertexType iris$overrideVertexType(ChunkVertexType vertexType) {
         // TODO: Don't use the extended vertex format when shaders are disabled.
-        return IrisModelVertexFormats.MODEL_VERTEX_XHFP;
+        return Iris.isPackActive() ? IrisModelVertexFormats.MODEL_VERTEX_XHFP : DefaultModelVertexFormats.MODEL_VERTEX_HFP;
```

Comment:
We can just return the `vertexType` if the pack is inactive here. Also, you can probably remove that TODO.